### DISCUSSION TITLE
Support for Performance Test

### DISCUSSION THREAD

**Discussion by dangkhoaphung** [webauthn4j/webauthn4j#1058] [Q&A] [answered]
Hi @ynojima ,
We would like to do performance test. In that case, how can we disable signature validation for both attestation and assertion flow?
Or do you have any idea to do performance test?
Could you help to advise.

Thanks

**Comment by ynojima** [comment, marked as answer]
In the webauthn processing, signature verification is an essential step that cannot be skipped.
I think performance test should also measure performance including signature verification.
